### PR TITLE
Fix unterminated quoted string bug

### PR DESCRIPTION
**Pull request by spikhoff**
See #21

### CODE REVIEW

File: code.sh [hunk 1]
```diff
@@ -298,7 +298,7 @@ fi
 # different toolboxes step on each other.
 
 homevar="$(flatpak-spawn --host podman inspect "$container_name" \
-    --format='{{ range .Config.Env }}{{ . }}{{"\n"}}{{ end }}' \
+    --format='{{ range .Config.Env }}{{ . }}\n{{ end }}' \
```
Comment (owtaylor) [inline code.sh:301]: If I try this with podman-3.4.4 from Fedora 35, I simply get literal "\n" in the output instead of newlines.

**Comment by owtaylor** [comment]
The test failures seem to mostly (completely?) be because the tests have various hardcoded expectations on the podman command and haven't been updated.

File: code.sh (hunk 1 above)
Comment (spikhoff) [reply code.sh]: I tested with 35 and updated pull-request. See how does it now.

**Comment by owtaylor** [comment]
I'd really like a bug filed against libpod and to see their response before I proceed here, We can work around bugs in podman, but I don't want to blindly work around bugs in libpod.

**Comment by spikhoff** [comment]
Fixed bracket typo, should be OK now.

**Pull request closed without merging**